node/src/test/scala/com/wavesplatform/http/AssetsRouteSpec.scala

```diff
@@ -309,4 +393,30 @@ class AssetsRouteSpec
     (response \ "minSponsoredAssetFee").asOpt[Long] shouldBe empty
     (response \ "originTransactionId").as[String] shouldBe tx.id().toString
   }
+
+  private def checkResponse(tx: InvokeExpressionTransaction, desc: AssetDescription, response: JsObject): Unit = {
```

Comment:
You can use the more convenient single call comparison with `responseAs[JsObject] should matchJson("{ ... }")`
See `com.wavesplatform.utils.JsonMatchers`